packages/medusa/src/api/routes/admin/payment-collections/create-payment-collection.ts

```diff
@@ -0,0 +1,58 @@
+import {
```

Comment:
thought: we should not allow admins to create payment collections without associating them with something (at least not initially).

For example, the first use case for payment collections will be to obtain payments from an OrderEdit - so would suggest that we initially only allow creation of payment collections when requesting an OrderEdit.